orga/app/services/url.js

```diff
@@ -12,4 +12,9 @@ export default class Url extends Service {
       this.definedCampaignsRootUrl :
       `${this.pixAppUrlWithoutExtension}${this.currentDomain.getExtension()}/campagnes/`;
   }
+
+  get homeUrl() {
+    return ENV.rootURL;
```

Comment:
A première lecture, j'ai l'impression qu'on ne peut pas gérer que la déconnexion sur app.pix.org renvoie sur pix.org. On va avoir besoin de `this.currentDomain.getExtension()`

Reply:
Euh.. là, on est sur PixOrga. Es-tu sûre que cela s'applique ?

Reply:
Oui pardon, j'ai noté app.pix.org, mais il y a aussi l'utilisation de orga.pix.org, dont l'url a commencé à être partagé je crois (à vérif coté team prescription)

Reply:
Corrigé !